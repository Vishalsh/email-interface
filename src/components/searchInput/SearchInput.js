import React from 'react';

import InputWithLabel from 'components/inputWithLabel/InputWithLabel'
import Button from 'components/button/Button'
import classes from './SearchInput.module.scss';

const SearchInput = () => {
  return (
    <div className="flex middle-xs">
      <div className={`${classes.inputWithLabelContainer} col-sm-8 col-md-9`}>
        <InputWithLabel label={''}
                        onChange={() => {
                        }}
                        value={''}
                        placeholder="Search email"
                        className={classes.inputWithLabel}/>
      </div>
      <Button type="primary" className="col-sm-4 col-md-3">
        Search
      </Button>
    </div>
  );
};

export default SearchInput;
