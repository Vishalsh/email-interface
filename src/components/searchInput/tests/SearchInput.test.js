import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import SearchInput from '../SearchInput';

describe('SearchInput', () => {
  it('should render the component', () => {
    const component = shallow(<SearchInput />);

    expect(toJson(component)).toMatchSnapshot();
  });
});