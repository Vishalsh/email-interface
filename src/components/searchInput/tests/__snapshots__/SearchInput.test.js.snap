// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchInput should render the component 1`] = `
<div
  className="row middle-xs"
>
  <div
    className="inputWithLabelContainer col-sm-8 col-md-9"
  >
    <InputWithLabel
      className="inputWithLabel"
      inputType="text"
      label=""
      onChange={[Function]}
      placeholder="Search email"
      textarea={false}
      value=""
    />
  </div>
  <Button
    className="col-sm-4 col-md-3"
    disabled={false}
    type="primary"
  >
    Search
  </Button>
</div>
`;
